assets/html/js/metadata.js

```diff
@@ -0,0 +1,11 @@
+// libraries: jquery
+// arguments: $
+
+let meta = $("div[data-docstringscollapsed]").data();
+
+if (meta.docstringscollapsed) {
+  $("#documenter-article-toggle-button").trigger({
+    type: "click",
+    noToggleAnimation: true,
+  });
+}
```

Comment:
This whole thing probably needs to be in a `$(document).ready(function () { ... }`?

Reply:
Let's try it! https://documenter.juliadocs.org/previews/PR2394/ should now have that change. Of course, all I can say that it continues to work for me, so you'll have to let us know :-)